board/static/board/css/basic.css

```diff
@@ -0,0 +1,100 @@
+.search_form{
```

Comment:
I guess this should be `search-form`. Also you can read about css methodology called BEM here: https://ru.bem.info/methodology/quick-start/